### ISSUE TITLE
new rule: avoid comma separated variable declarations

### ISSUE THREAD

**Issue by HamletDRC** [Microsoft/tslint-microsoft-contrib#25]
new rule: avoid comma separated variable declarations

**Comment by loicraux** [comment]
Should multiple variable declarations be allowed in for statements though?

Such as :
```javascript
for (var i = 0, j = 1; i < j; i++) { ... }
```

- I know that `iterable.forEach()` is also preferred over for loops
- and fyi, [JSCS allows these multiple var declarations within a for loop by default](http://jscs.info/rule/disallowMultipleVarDecl).

**Comment by HamletDRC** [comment]
Yes, I think the comma separate list in for-loop should be allowed.
